devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup.yaml

```diff
@@ -15,13 +15,18 @@ spec:
     - hosts: all
       strategy: linear
       tasks:
+        - name: Install Python dependencies
+          ansible.builtin.package:
+            name:
+              - git
+              - python-setuptools
+              - python-requests
+              - python3-pip
+            state: present
+        - name: Install repo-setup directly from github
+          ansible.builtin.pip:
+            name: git+https://github.com/openstack-k8s-operators/repo-setup
```

Comment:
become: true? Or is it not needed?

Reply:
I was a wondering about that too. But it appears that the user executing these is already given all permissions necessary. And the standard deployment runs as expected. Still, just in case the situation changes I can have it future proofed.